### PR TITLE
Made search case insensitive

### PR DESCRIPTION
**Pull request by Hetarth02**
This PR aims to fix Issue #2373.

image

### CODE REVIEW

**Comment by Hetarth02** [comment]
@mortenpi  Before merging, where do I add the entry in `CHANGELOG.md`? In version 1.2.1 or any other place?

**Comment by fredrikekre** [comment]
Vim has `smartcase` which I have started to like. Basically, if the search query contains at least one upper case letter, then the search is case sensitive, otherwise not. Perhaps that is something to mimic here?

**Comment by fredrikekre** [comment]
> where do I add the entry in CHANGELOG.md? In version 1.2.1 or any other place?

Above 1.2.1 in a new header with `### Unreleased`

**Comment by LilithHafner** [comment]
Honestly, this seems like an upstream bug in mini-search. I would hope that even a very light fuzzy search would fix this, but apparently they use Levenshtein edit distance to compute fuzzy matches which is not a case-aware system (i.e. Editlink and editlink are equally far apart as editWink and editlink).

source: https://lucaong.github.io/minisearch/types/MiniSearch.SearchOptions.html#__type.weights.__type-10.fuzzy-1

I think smartcase seems like a reasonable idea and it's also worth trying out `fuzzy: true` to see if minisearch's default fuzzy parameters are enough to fix this.

**Review by LilithHafner — APPROVED** [review]
This seems like an unambiguous improvement. LGTM.

Smartcase is certainly an interesting idea to try out, but does not block this PR.

File: CHANGELOG.md
Comment (mortenpi) [inline CHANGELOG.md:8]:
```suggestion
* The search in the HTML output is now case-insensitive. ([#2373], [#2374])
```

**Review by mortenpi — APPROVED** [review]
LGTM as well, sorry for the delay. Thank you!

File: CHANGELOG.md
Comment (mortenpi) [inline CHANGELOG.md:6]:
```suggestion
## Unreleased

### Changed
```

**Pull request merged**
